packages/engine/Source/Core/createWorldBathymetryAsync.js

```diff
@@ -0,0 +1,48 @@
+import CesiumTerrainProvider from "./CesiumTerrainProvider.js";
+import defaultValue from "./defaultValue.js";
+
+/**
+ * Creates a {@link CesiumTerrainProvider} instance for the {@link https://cesium.com/content/#cesium-world-bathymetry | Cesium World Bathymetry}.
```

Comment:
```suggestion
 * Creates a {@link CesiumTerrainProvider} instance for the {@link https://cesium.com/content/#cesium-world-bathymetry|Cesium World Bathymetry}.
```

For some reason this adds the percent encoded space onto the url, breaking the permalink.

Reply:
Interestingly, removing the spaces creates the opposite problem where the pipe and next word are considered part of the link in intellisense and viscose. I was pretty sure I validated these worked fine in the generated docs with no issues, where did you see the broken links?

Reply:
It's really annoying VSCode/intellisense doesn't match the parsing behavior but the public docs are more important, I've reverted.

![2024-01-30_11-20](https://github.com/CesiumGS/cesium/assets/8007967/a8d6fba5-5996-4e29-b027-cbca9049df87)

Reply:
Just cause I couldn't help digging into this. JSDoc is supposed to trim spaces on both the link and label per this commit 10 months ago https://github.com/jsdoc/jsdoc/commit/67d043160fead32d8ffdcd28ef5cf9ec50bac92c
We might just need to update jsdoc but that can be a separate issue.